cmd/version.go

```diff
@@ -20,11 +26,60 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Maybe it's just me but I think it reads easier if this is a named function, like [here](https://github.com/exercism/configlet/blob/master/cmd/lint.go#L22)